src/main/java/gov/nasa/cumulus/CNMResponse.java

```diff
@@ -58,19 +58,30 @@ public void handleRequestStreams(InputStream inputStream, OutputStream outputStr
 	public static JsonObject getResponseObject(String exception) {
 		JsonObject response = new JsonObject();
 
-		if(exception == null || new String("").equals(exception) ||  new String("None").equals(exception) ||  new String("\"None\"").equals(exception)){
-			//success
+        if (exception == null || exception.isEmpty() || exception.equals("None") || exception.equals("\"None\"")) {
```

Comment:
use apache lang3 StringUtils class which is null safe
it has trim(), equals ... static functions